err_fmt_test.go

```diff
@@ -856,3 +856,77 @@ func TestFmt(t *testing.T) {
 		})
 	}
 }
+
+func TestErrCore_String(t *testing.T) {
+	table := []struct {
+		name        string
+		core        *clues.ErrCore
+		expectS     string
+		expectVPlus string
+	}{
+		{
+			name: "all values",
+			core: clues.
+				New("message").
+				With("key", "value").
+				Label("label").
+				Core(),
+			expectS:     `{"message", [label], {key:value}}`,
+			expectVPlus: `{msg:"message", labels:[label], values:{key:value}}`,
+		},
+		{
+			name: "message only",
+			core: clues.
+				New("message").
+				Core(),
+			expectS:     `{"message"}`,
+			expectVPlus: `{msg:"message", labels:[], values:{}}`,
+		},
+		{
+			name: "labels only",
+			core: clues.
+				New("").
+				Label("label").
```

Comment:
Maybe add a test with multiple labels?

Reply:
that's more chaff than value in this case, I think.  Labels are pretty thoroughly tested otherwise.